### ISSUE TITLE
How add new language 

### ISSUE THREAD

**Issue by hamedkhosrawi** [cerberustesting/cerberus-core#2468]
Hi, first of all I want to thanks to you and please guide me how can I add a new language a RTL language to Cerberus? I want add my language which is Persian to Cerberus app

**Comment by vertigo17** [comment]
Hello,
All documentation texts are located into:
 cerberus-core/source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java

For example if I want to add a Persian language to cerberus I will have to add after:
        b.append("('application','Application','','en','Application','','_application_attributes')");
        b.append(",('application','Application','','ru','Приложение','','_application_attributes')");
        b.append(",('application','Application','','fr','Application','','_application_attributes')");

1 extra line that will look like:
        b.append("('application','Application','','**fa**','کاربرد','','_application_attributes')");

If you are interested to perform that work, I will also add the Persian entry inside the combo box that select the language. Just tell me if you want to proceed ;-).
That would be a nice contribution.

**Comment by hamedkhosrawi** [comment]
yes of course, because cerberus platform include so mush details about functional testing and manual testing for web application and desktop , so i want to add my language.

**Comment by vertigo17** [comment]
Ok so I let you create some pull request with the first text you whant to translate. I will review them and and the language combo when done.

**Comment by hamedkhosrawi** [comment]
i reached line 852 in DocumentationDataBaseService, i wait until it's getting finished or i create pull request?

**Comment by vertigo17** [comment]
That was quite fast ;-)
Please create a PR progressively. I will add the language at the same time so that we can preview the result on qa.cerberus-testing.com

**Comment by hamedkhosrawi** [comment]
i tried to make PR but i'm getting 403 forbidden error, how can i create pull request?

**Comment by vertigo17** [comment]
I added you in the team in read mode.
In theory, that should not be necessary.
In fact you should fork the rep to your user, comit from your local repo and create a Pull Request (PR) from there.

**Comment by hamedkhosrawi** [comment]
hi, i create pull request for parsian-lang, Thank you for checking

**Comment by vertigo17** [comment]
I reviewed and merged your PR. It was quite good !!!
Thanks a lot for this contribution.
I commit a couple of fixes and added the main invariant so that we can test the result on qa.cerberus-testing.com.
Mainly you forgot to change en to fa adding some duplicates entries ;-).
I would suggest to prioritize the page_header and page_global sections as they are used on almost all screens.
Don't forget to pull my changes before continuing.

**Comment by hamedkhosrawi** [comment]
thnak you very mush, i appologize about duplicate lines which bothered you, i pull your changes and keeping translation on en lines.

**Comment by vertigo17** [comment]
No problem and thanks again for this contribution.
